### PR TITLE
apply uncontrolled component input

### PR DESCRIPTION
**Pull request by tranchinh123**

### CODE REVIEW

File: bookshelter-app/package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "react-hook-form": "^7.53.1",
```
Comment (huypham-agilityio) [inline bookshelter-app/package.json]: do you use this?

File: bookshelter-app/src/components/CategoriesSection/index.tsx [hunk 2]
```diff
@@ -5,31 +5,41 @@ import { Category } from '../../types';
...
   const [categories, setCategories] = useState<Category[]>([]);
   const { slug, id } = useParams<{ slug: string; id: string }>();
   const [searchParams] = useSearchParams();
+  const { showToast } = useToast();
 
   const name = searchParams.get('query');
   const location = useLocation();
   const isHomePage = location.pathname === '/';
 
+  console.log('bbb');
```
Comment (huypham-agilityio) [inline bookshelter-app/src/components/CategoriesSection/index.tsx:20]: remove log

Comment (huypham-agilityio) [reply bookshelter-app/src/components/CategoriesSection/index.tsx:20]: For the next PR, pls review code changed before submitting

File: bookshelter-app/src/components/CategoryList/index.tsx [hunk 3]
```diff
@@ -32,6 +32,7 @@ const CategoryList = ({
       setIsMenuOpen(false);
     }
   };
+  console.log('bbb');
```
Comment (huypham-agilityio) [inline bookshelter-app/src/components/CategoryList/index.tsx:35]: same here

Comment (tranchinh123) [reply bookshelter-app/src/components/CategoryList/index.tsx:35]: Fixed!

File: bookshelter-app/src/components/CategoriesSection/index.tsx (hunk 2 above)
Comment (tranchinh123) [reply bookshelter-app/src/components/CategoriesSection/index.tsx:20]: Fixed!

File: bookshelter-app/package.json (hunk 1 above)
Comment (tranchinh123) [reply bookshelter-app/package.json]: Fixed!

**Pull request merged**